application/src/main/java/com/sanction/thunder/resources/VerificationResource.java

```diff
@@ -39,13 +46,100 @@
    * @param metrics The metrics object to set up meters with.
    */
   @Inject
-  public VerificationResource(PilotUsersDao usersDao, MetricRegistry metrics) {
+  public VerificationResource(PilotUsersDao usersDao,
+                              MetricRegistry metrics,
+                              EmailService emailService) {
     this.usersDao = usersDao;
+    this.emailService = emailService;
 
     // Set up metrics
-    this.postRequests = metrics.meter(MetricRegistry.name(
+    this.getRequests = metrics.meter(MetricRegistry.name(
         UserResource.class,
-        "post-requests"));
+        "get-requests"));
+  }
+
+  /**
+   * Validates a user account by sending an email with a unique token.
+   *
+   * @param key The basic authentication key necessary to access the resource.
+   * @param email The email to send a unique token to.
+   * @return A response status and message.
+   */
+  @POST
+  public Response verifyUser(@Auth Key key,
+                             @QueryParam("email") String email,
+                             @HeaderParam("password") String password) {
+    getRequests.mark();
+
+    if (email == null || email.isEmpty()) {
+      LOG.warn("Attempted user verification without an email.");
+      return Response.status(Response.Status.BAD_REQUEST)
+          .entity("Incorrect or missing email query parameter.").build();
+    }
+
+    if (password == null || password.isEmpty()) {
+      LOG.warn("Attempted to verify user {} without a password.", email);
+      return Response.status(Response.Status.BAD_REQUEST)
+          .entity("Incorrect or missing header credentials.").build();
+    }
+
+    LOG.info("Attempting to verify user {}", email);
```

Comment:
Can we make this say "Attempting to send verification email to user {}"?

Reply:
We sure can!